project/script_VMRCM.tcl

```diff
@@ -8,54 +8,7 @@ source env_hls.tcl
 
 # list of modules to test in C-simulation
 set modules_to_test {
-  {VMRCM_L1PHIA}
-  {VMRCM_L1PHIB}
-  {VMRCM_L1PHIC}
-  {VMRCM_L1PHID}
-  {VMRCM_L1PHIE}
-  {VMRCM_L1PHIF}
-  {VMRCM_L1PHIG}
-  {VMRCM_L1PHIH}
   {VMRCM_L2PHIA}
```

Comment:
@aryd Could you please add back the other VMRCMs for the benefit of the CI?
```suggestion
  {VMRCM_L1PHIA}
  {VMRCM_L1PHIB}
  {VMRCM_L1PHIC}
  {VMRCM_L1PHID}
  {VMRCM_L1PHIE}
  {VMRCM_L1PHIF}
  {VMRCM_L1PHIG}
  {VMRCM_L1PHIH}
  {VMRCM_L2PHIA}
  {VMRCM_L2PHIB}
  {VMRCM_L2PHIC}
  {VMRCM_L2PHID}
  {VMRCM_L3PHIA}
  {VMRCM_L3PHIB}
  {VMRCM_L3PHIC}
  {VMRCM_L3PHID}
  {VMRCM_L4PHIA}
  {VMRCM_L4PHIB}
  {VMRCM_L4PHIC}
  {VMRCM_L4PHID}
  {VMRCM_L5PHIA}
  {VMRCM_L5PHIB}
  {VMRCM_L5PHIC}
  {VMRCM_L5PHID}
  {VMRCM_L6PHIA}
  {VMRCM_L6PHIB}
  {VMRCM_L6PHIC}
  {VMRCM_L6PHID}
  {VMRCM_D1PHIA}
  {VMRCM_D1PHIB}
  {VMRCM_D1PHIC}
  {VMRCM_D1PHID}
  {VMRCM_D2PHIA}
  {VMRCM_D2PHIB}
  {VMRCM_D2PHIC}
  {VMRCM_D2PHID}
  {VMRCM_D3PHIA}
  {VMRCM_D3PHIB}
  {VMRCM_D3PHIC}
  {VMRCM_D3PHID}
  {VMRCM_D4PHIA}
  {VMRCM_D4PHIB}
  {VMRCM_D4PHIC}
  {VMRCM_D4PHID}
  {VMRCM_D5PHIA}
  {VMRCM_D5PHIB}
  {VMRCM_D5PHIC}
  {VMRCM_D5PHID}
```

Reply:
Done